backend/application/syson-application/src/test/java/org/eclipse/syson/application/controllers/diagrams/general/view/GVTopNodeCreationTests.java

```diff
@@ -155,42 +156,42 @@ public class GVTopNodeCreationTests extends AbstractIntegrationTests {
-                Arguments.of(SysmlPackage.eINSTANCE.getPackage(), 0),
-                Arguments.of(SysmlPackage.eINSTANCE.getPartDefinition(), 9),
-                Arguments.of(SysmlPackage.eINSTANCE.getPartUsage(), 10),
-                Arguments.of(SysmlPackage.eINSTANCE.getPortDefinition(), 4),
-                Arguments.of(SysmlPackage.eINSTANCE.getPortUsage(), 4),
-                Arguments.of(SysmlPackage.eINSTANCE.getRequirementDefinition(), 8),
-                Arguments.of(SysmlPackage.eINSTANCE.getRequirementUsage(), 8),
-                Arguments.of(SysmlPackage.eINSTANCE.getUseCaseDefinition(), 5),
-                Arguments.of(SysmlPackage.eINSTANCE.getUseCaseUsage(), 7),
-                Arguments.of(SysmlPackage.eINSTANCE.getSatisfyRequirementUsage(), 8),
-                Arguments.of(SysmlPackage.eINSTANCE.getStateDefinition(), 6),
-                Arguments.of(SysmlPackage.eINSTANCE.getStateUsage(), 6),
-                Arguments.of(SysmlPackage.eINSTANCE.getViewUsage(), 0)
+                Arguments.of(SysmlPackage.eINSTANCE.getPackage(), 0, "101", "200"),
+                Arguments.of(SysmlPackage.eINSTANCE.getPartDefinition(), 9, ViewConstants.DEFAULT_DEFINITION_NODE_HEIGHT, ViewConstants.DEFAULT_DEFINITION_NODE_WIDTH),
+                Arguments.of(SysmlPackage.eINSTANCE.getPartUsage(), 10, ViewConstants.DEFAULT_USAGE_NODE_HEIGHT, ViewConstants.DEFAULT_USAGE_NODE_WIDTH),
+                Arguments.of(SysmlPackage.eINSTANCE.getPortDefinition(), 4, ViewConstants.DEFAULT_DEFINITION_NODE_HEIGHT, ViewConstants.DEFAULT_DEFINITION_NODE_WIDTH),
+                Arguments.of(SysmlPackage.eINSTANCE.getPortUsage(), 4, ViewConstants.DEFAULT_USAGE_NODE_HEIGHT, ViewConstants.DEFAULT_USAGE_NODE_WIDTH),
+                Arguments.of(SysmlPackage.eINSTANCE.getRequirementDefinition(), 8, ViewConstants.DEFAULT_DEFINITION_NODE_HEIGHT, ViewConstants.DEFAULT_DEFINITION_NODE_WIDTH),
+                Arguments.of(SysmlPackage.eINSTANCE.getRequirementUsage(), 8, ViewConstants.DEFAULT_USAGE_NODE_HEIGHT, ViewConstants.DEFAULT_USAGE_NODE_WIDTH),
+                Arguments.of(SysmlPackage.eINSTANCE.getUseCaseDefinition(), 5, ViewConstants.DEFAULT_DEFINITION_NODE_HEIGHT, ViewConstants.DEFAULT_DEFINITION_NODE_WIDTH),
+                Arguments.of(SysmlPackage.eINSTANCE.getUseCaseUsage(), 7, ViewConstants.DEFAULT_USAGE_NODE_HEIGHT, ViewConstants.DEFAULT_USAGE_NODE_WIDTH),
+                Arguments.of(SysmlPackage.eINSTANCE.getSatisfyRequirementUsage(), 8, ViewConstants.DEFAULT_USAGE_NODE_HEIGHT, ViewConstants.DEFAULT_USAGE_NODE_WIDTH),
+                Arguments.of(SysmlPackage.eINSTANCE.getStateDefinition(), 6, ViewConstants.DEFAULT_DEFINITION_NODE_HEIGHT, ViewConstants.DEFAULT_DEFINITION_NODE_WIDTH),
+                Arguments.of(SysmlPackage.eINSTANCE.getStateUsage(), 6, ViewConstants.DEFAULT_USAGE_NODE_HEIGHT, ViewConstants.DEFAULT_USAGE_NODE_WIDTH),
+                Arguments.of(SysmlPackage.eINSTANCE.getViewUsage(), 0, "101", "300")
```

Comment:
Could you create constants for ViewUsage height & width please? And use them in ViewUsageNodeDescription? Thank you